.github/workflows/main.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         if: github.event_name != 'pull_request' && github.event.schedule != '30 5 * * 1-5'
         run: |
           echo "NTHREADS=64" >> $GITHUB_ENV
-          echo "MAX_FILES=-2" >> $GITHUB_ENV
+          echo "MAX_FILES=1" >> $GITHUB_ENV
```

Comment:
@davidwalter2 does this line really change the number of files in the histmaker?
It doesn't look like it

Reply:
I don't want to change it permanently, I just wanted to make a quick test of the CI but I just run the whole thing :)
I'll revert this in the next commit